packages/helpermodules/command.py

```diff
@@ -771,7 +771,8 @@ def __enter__(self):
     def __exit__(self, exception_type, exception, exception_traceback) -> bool:
         if isinstance(exception, Exception):
             pub_user_message(self.payload, self.connection_id,
-                             f'Es ist ein interner Fehler aufgetreten: {traceback.format_exc()}', MessageType.ERROR)
+                             f'Es ist ein interner Fehler aufgetreten: {exception}', MessageType.ERROR)
+            log.debug({traceback.format_exc()})
```

Comment:
```suggestion
            log.exception("Fehler bei der command-Ausführung)
```
Bitte ausprobieren, ob log.exception im `__exit__`-Logtext funktioniert, sonst die Exception mit log.error loggen, damit sie in allen Logleveln geloggt wird.

Reply:
> Bitte ausprobieren, ob log.exception im `__exit__`-Logtext funktioniert, sonst die Exception mit log.error loggen, damit sie in allen Logleveln geloggt wird.

Nein, wird nicht im __exit__ übergeben (Exception: None), es wird nur der traceback im log angezeigt, wenn  log.exception("Fehler beim Erstellen der Cloud-Sicherung: ") in command.py l.646 gesetzt wird